test/imath/ref/out.xml

```diff
@@ -0,0 +1,517 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<TranslationUnit name="/home/anders/code/cppmm/test/imath/bind/imath_box.cpp" id="5">
```

Comment:
is it worth grouping records by namespaces? In case in the future people want to extract that information for creating modules?

Reply:
I have a placeholder node for namespaces but haven't used it since there's a complex relationship with TUs - one TU might have many namespaces or one namespace might be spread over many TUs. I prefer the TU as a way of grouping classes into modules. If you want to reorganize at generator time based on the namespaces then they're there in the qualified names.